src/components/MenuButton.test.js

```diff
@@ -0,0 +1,15 @@
+import { renderComponent } from '../testsUtilities/renderComponent';
+import MenuButton from './MenuButton';
+
+describe('Menu button test', () => {
+  it('Should show renederd component', () => {
+    renderComponent(MenuButton('Test button', onclick));
+
+    const testButton = document.querySelector('.menuButton');
+    const testButtonInnerText = document.querySelector('.menuButton__text');
+
+    expect(testButton).toBeInTheDocument();
+    expect(testButtonInnerText).toBeInTheDocument();
+    expect(testButtonInnerText.innerText).toBe('Test button');
+  });
+});
```

Comment:
Powinien zostać dodany jeszcze test czy jak klikniesz przycisk, to czy metoda do niego przekazana zostanie wywołana.

Reply:
https://jestjs.io/docs/mock-functions